osbuild/util/ostree.py

```diff
@@ -215,15 +214,52 @@ def parse_input_commits(commits):
 def deployment_path(root: PathLike, osname: str, ref: str, serial: int):
     """Return the path to a deployment given the parameters"""
 
-    base = os.path.join(root, "ostree")
+    if osname == "" and ref == "" and serial == None:
+        filenames =  glob.glob(root + '/ostree/deploy/*/deploy/*.0', recursive=True)
+        if len(filenames) < 1:
+            raise ValueError("Cound not find deployment")
+        elif len(filenames) > 1:
+            raise ValueError("More than one deployment found")
+        return filenames[0]
+    
+    else:
```

Comment:
Minor/optional: since we're returning in the true case, we can drop this else and save one indentation